web/package.json

```diff
@@ -53,7 +53,7 @@
     "eslint-plugin-tailwindcss": "latest",
     "jsdom": "20.0.1",
     "postcss": "8.4.18",
-    "prettier": "2.7.1",
+    "prettier": "^2.5.1",

```

Comment:
What happened here 👀? Any reason why we are downgrading prettier?

Reply:
No reason oops let me fix that